ui/app/components/auth-config-form/options.js

```diff
@@ -8,61 +8,63 @@ import AuthConfigComponent from './config';
 import { service } from '@ember/service';
 import { task } from 'ember-concurrency';
 import { waitFor } from '@ember/test-waiters';
+import { tracked } from '@glimmer/tracking';
+import errorMessage from 'vault/utils/error-message';
 
 /**
  * @module AuthConfigForm/Options
  * The `AuthConfigForm/Options` is options portion of the auth config form.
  *
  * @example
  * ```js
- * {{auth-config-form/options model.model}}
+ * <AuthConfigForm::Options @modle={{this.args.model.model}} />
  * ```
  *
  * @property model=null {DS.Model} - The corresponding auth model that is being configured.
  *
  */
 
-export default AuthConfigComponent.extend({
-  flashMessages: service(),
-  router: service(),
+export default class AuthConfigOptions extends AuthConfigComponent {
+  @service flashMessages;
+  @service router;
 
-  saveModel: task(
-    waitFor(function* () {
-      const data = this.model.config.serialize();
-      data.description = this.model.description;
-      data.user_lockout_config = {};
+  @tracked errorMessage;
 
-      // token_type should not be tuneable for the token auth method.
-      if (this.model.methodType === 'token') {
-        delete data.token_type;
-      }
+  @task
+  @waitFor
+  *saveModel(evt) {
+    evt.preventDefault();
+    this.error = null;
```

Comment:
i think this should be 
```suggestion
    this.errorMessage = null;
```